### DISCUSSION TITLE
Checkpoint for trained models

### DISCUSSION THREAD

**Discussion by parthnatekar** [tensorflow/compression#107] [General]
Hi, where can I find the checkpoints for the trained models.

According to documentation [here](https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/import_meta_graph), you need to load the graph using the saved GraphDef and then restore the checkpoints. I can't seem to find where the checkpoints are being restored.

I need to run the evaluate.py script on the trained models.

**Comment by jonaballe** [comment]
Hi there, checkpoints are not necessary. The metagraphs that we publish (see `tfci.py` for code to load/run them) have all the variables baked in as constants.
